docs/source/using/managing-minishift.adoc

```diff
@@ -53,6 +53,41 @@ the GitHub issue https://github.com/minishift/minishift/issues/179[#179].
 The link:../command-ref/minishift_delete{outfilesuffix}[`minishift delete`] command deletes the OpenShift cluster,
 and also shuts down and deletes the Minishift VM. No data or state are preserved.
 
+[[choosing-iso-image-for-vm]]
+== Choosing the ISO image for the VM
+
+When you start Minishift, it downloads a live ISO image that the hypervisor uses to provision the host VM.
+This VM then locally provisions the OpenShift cluster.
+
+The following ISO images are available:
+
+- link:https://github.com/minishift/minishift-b2d-iso/releases[Boot2Docker] (Default). This ISO is
+based on the link:https://github.com/boot2docker/boot2docker[Boot2Docker] ISO, which is a
+lightweight Linux distribution optimized for containerized application development.
+
+- link:https://github.com/minishift/minishift-centos-iso/releases[CentOS]. This ISO contains
+the link:https://www.centos.org/[CentOS] operating system, which is an enterprise-ready Linux
+distribution.
++
+NOTE: The CentOS ISO image size is larger than the Boot2Docker image, because CentOS is a
```

Comment:
@thatdocslady The idea here is to tell users about the sizes of the ISO and also the advantage of both the ISO images so that user can take a conscious decision. When we compare both the ISOs , boot2docker ISO size is a lot less than CentOS. Hence users who are conscious about the size would go for boot2docker. However the downside is, Boot2Docker is not a Linux distribution which can be run in production. CentOS on the other hand  runs in production i.e. enterprise Linux/GNU distribution.  So user who want to have development environment (i.e. OpenShift on local workstation) similar to the production would opt for CentOS, even if it is higher size. Hope this clarify things.